documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -8,51 +8,103 @@ weight: 4
+- [Running and Licensing Oracle Programs in Containers and Kubernetes](https://www.oracle.com/a/tech/docs/running-and-licensing-programs-in-containers-and-kubernetes.pdf)
 
 Some supported environments have additional help or samples that are specific
 to the operator, or are subject to limitations and restrictions: see
 [Important notes about specific environments](#important-notes-about-specific-environments).
 
-### Kubernetes, WebLogic, and operating system prerequisites
+### Kubernetes, WebLogic Server, and operating system prerequisites
 
-The operator is subject to Kubernetes, WebLogic, and operating system versioning prerequisites:
+The operator is subject to Kubernetes, WebLogic Server, and operating system versioning prerequisites:
 see [Operator prerequisites]({{< relref "/userguide/prerequisites/introduction.md" >}}).
 
 ### Pricing and licensing
 
+The WebLogic Kubernetes Operator and Oracle Linux are open source and free;
+WebLogic Server requires licenses unless used in a single developer desktop development environment.
+In detail:
+
+#### WebLogic Kubernetes Operator
+
 The WebLogic Kubernetes Operator (the "operator") is open source and free,
 licensed under the Universal Permissive license (UPL), Version 1.0.
+For support details, see [Get help]({{< relref "userguide/introduction/get-help.md" >}}).
+
+#### WebLogic Server
 
 WebLogic Server is not open source.
 Licensing is required for each running WebLogic Server instance in Kubernetes,
 just as with any deployment of WebLogic Server.
 Licensing is free for a single developer desktop development environment.
+For more information, see the [Fusion Middleware Licensing Information User Manual - Application Server Products](https://docs.oracle.com/en/middleware/fusion-middleware/fmwlc/application-server-products-new-structure.html).
+
+#### Oracle Linux and WebLogic Server Images
+
+Oracle Linux is under open source license
+and is completely free to download and use.
+
+In addition, with WebLogic license and support,
+customers have access to:
+- The latest WebLogic Server images
+  which bundle Java SE and the latest slim Oracle Linux images.
+- Oracle Linux support.
+Note that WebLogic licenses and support do not include customer entitlements
+for direct access to Oracle Linux support or Unbreakable Linux Network
+(to access the standalone Oracle Linux patches).
+
+Oracle supplies two types of WebLogic Server images, patched (CPU) and unpatched.
+New WebLogic Server images are released when security fixes
+are released for Oracle Linux, WebLogic Server, or Java. In addition,
+patched images are rebuilt every CPU cycle, which is quarterly.
+Customer support for WebLogic Server images is handled by Oracle support.
+
+Using the [WebLogic Image Tool]("https://github.com/oracle/weblogic-image-tool")
```

Comment:
("https://github.com/oracle/weblogic-image-tool") - not sure that you need the quotation marks inside the parentheses.

Reply:
fixed